### ISSUE TITLE
The HyperV provider can't resize the host disk

### ISSUE THREAD

**Issue by petr-motejlek** [petr-motejlek/elk#8]

**Comment by petr-motejlek** [comment]
Could instead I use VirtualBox (since it started supporting HyperV paravirtualization recently) to spin up a control machine and several nodes, and then use them?

**Comment by petr-motejlek** [comment]
It's finally not hard to run VirtualBox side by side with HyperV (WSL). And since VirtualBox runs on more platforms, I'd prefer to support VirtualBox instead of HyperV. Let's do it.

One thing I noticed when playing with VirtualBox is that all of a sudden (I swear it worked weeks ago) port forwarding onto host ports such as 2222 does not work. VirtualBox's log just contains one useless error: "NAT: Failed to redirect TCP", which lead me to https://stackoverflow.com/a/60396146/4831064 and because of that I have switched from 2222 as the default SSH host port to 22000.

```
config.vm.usable_port_range = 22000..22222
config.vm.network :forwarded_port, guest: 22, host: 2222, disabled: true
config.vm.network :forwarded_port, guest: 22, host: 22000, id: "ssh"
```
